### PR TITLE
Dionae Ship Improvements 

### PR DESCRIPTION
**Pull request by Yonnimer**
Changes the layout of both ships slightly to make them less of a pain for slowbois to get around, as well as adds netguns to both ships.

Also fixes both ship icons being invisible

### CODE REVIEW

**Comment by Yonnimer** [comment]
!review

**Review by NonQueueingMatt — APPROVED** [review]

**Comment by Yonnimer** [comment]
As far as i'm aware, the conflict is something to do with the checker, not something wrong with the pr itself, so it should still be fine to review/merge

**Review by alsoandanswer — APPROVED** [review]

File: maps/away/ships/dionae/diona_serz_ship/diona_serz_ship_ghostroles.dm [hunk 1]
```diff
@@ -7,7 +7,7 @@
 	tags = list("External")
 
 	spawnpoints = list("serz_voidtamer")
-	max_count = 2
```
Comment (Arrow768) [inline maps/away/ships/dionae/diona_serz_ship/diona_serz_ship_ghostroles.dm:10]: Undocumented change

File: maps/away/ships/dionae/diona_rokz_ship/diona_rokz_ship_ghostroles.dm [hunk 2]
```diff
@@ -7,7 +7,7 @@
 	tags = list("External")
 
 	spawnpoints = list("rokz_voidtamer")
-	max_count = 2
```
Comment (Arrow768) [inline maps/away/ships/dionae/diona_rokz_ship/diona_rokz_ship_ghostroles.dm:10]: undocumented change

**Review by Arrow768 — CHANGES_REQUESTED** [review]

File: maps/away/ships/dionae/diona_serz_ship/diona_serz_ship_ghostroles.dm (hunk 1 above)
Comment (Yonnimer) [reply maps/away/ships/dionae/diona_serz_ship/diona_serz_ship_ghostroles.dm:10]: Changelog has been updated

File: maps/away/ships/dionae/diona_rokz_ship/diona_rokz_ship_ghostroles.dm (hunk 2 above)
Comment (Yonnimer) [reply maps/away/ships/dionae/diona_rokz_ship/diona_rokz_ship_ghostroles.dm:10]: Changelog has been updated

**Pull request merged**